rust/examples/openvasd/config.example.toml

```diff
@@ -10,16 +10,19 @@ enable_get_scans = true
 key = "mtls_is_preferred"
 
 [tls]
-certs = "/etc/sensord/tls/certs.pem"
-key = "/var/lib/sensord/tls/key.pem"
-client_certs = "/etc/sensord/tls/clients"
+certs = "/etc/openvasd/tls/server.pem"
+key = "/var/lib/openvasd/tls/server.pem"
+client_certs = "/etc/openvasd/tls/client"
 
 [ospd]
-socket = "/var/run/ospd/ospd.sock"
+socket = "/var/run/ospd/ospd-openvas.sock"
 
 [ospd.result_check_interval]
 secs = 1
 nanos = 0
 
 [listener]
 address = "127.0.0.1:3000"
+
+[log]
+level = "DEBUG"
```

Comment:
Currently the log level cannot be parsed with the config file. I spent some time investigating, with no clue, why it does not work